awx/main/tests/functional/models/test_events.py

```diff
@@ -45,69 +49,15 @@ def test_parent_failed(emit, event):
         job_id=j.pk,
         parent_uuid='abc123',
         event=event
-    )
-    assert JobEvent.objects.count() == 2
-    for e in JobEvent.objects.all():
-        assert e.failed is True
-
-
-@pytest.mark.django_db
-@mock.patch('awx.main.consumers.emit_channel_notification')
-def test_job_event_websocket_notifications(emit):
```

Comment:
These websocket emits don't actually occur via signals anymore, so there's no need to test that side effect (also, this behavior has fairly adequate coverage elsewhere in integration testing).